commit 13e9b27f5f9550b825128f8cd1f31d20dc8fa91b (HEAD, origin/upgrade-ppxlib-0.18.0)
Author: Nathan Rebours <nathan.p.rebours@gmail.com>
Date:   Mon Oct 5 17:33:56 2020 +0200

    Make ppx_sexp_conv compatible with ppxlib.0.18.0
    
    ppxlib.0.18.0 upgrades to the 4.11 AST which results in a change
    in string constants representation. This PR makes ppx_sexp_conv
    compatible with the latest ppxlib.
    
    You might want for the actual release of ppxlib.0.18.0 before merging
    this!
    
    Signed-off-by: Nathan Rebours <nathan.p.rebours@gmail.com>

diff --git a/expander/sexp_grammar_lifter.ml b/expander/sexp_grammar_lifter.ml
index 05005bf..17f1e73 100644
--- a/expander/sexp_grammar_lifter.ml
+++ b/expander/sexp_grammar_lifter.ml
@@ -28,7 +28,7 @@ type atom      = Atom.t
 type var_name  = Sexp.Private.Raw_grammar.var_name
 type type_name = Sexp.Private.Raw_grammar.type_name
 
-let lift_string ~loc s = pexp_constant ~loc (Pconst_string (s, None))
+let lift_string ~loc s = pexp_constant ~loc (Pconst_string (s, loc, None))
 let lift_var_name      = lift_string
 let lift_type_name     = lift_string
 
diff --git a/ppx_sexp_conv.opam b/ppx_sexp_conv.opam
index 06fd03e..05c4182 100644
--- a/ppx_sexp_conv.opam
+++ b/ppx_sexp_conv.opam
@@ -15,7 +15,7 @@ depends: [
   "base"     {>= "v0.14" & < "v0.15"}
   "sexplib0" {>= "v0.14" & < "v0.15"}
   "dune"     {>= "2.0.0"}
-  "ppxlib"   {>= "0.11.0"}
+  "ppxlib"   {>= "0.18.0"}
 ]
 synopsis: "[@@deriving] plugin to generate S-expression conversion functions"
 description: "
